integr8ly/installation
# INTLY-615 Include playbook for the application monitoring operator

Currently, the application monitoring operator is not included in
the installer.

This change includes the application monitoring operator in the
installer, including a role for the application monitoring
operator and a standalone playbook.

Verification:
- Run the playbooks/install_application_monitoring.yml playbook
against a cluster. Ensure the operator is setup correctly.
- Run the playbook again and ensure it is skipped.
- Run the playbooks/install.yml playbook and ensure the monitoring
playbook is executed too.
- Setting application_monitoring to false in the manifest should
force skipping the run of the playbook.
